Core.Collectors.Tests/Web/FixedHttpClient.cs

```diff
@@ -132,5 +140,10 @@ public Task<HttpResponseMessage> PostAsync(string requestUrl, IAuthentication au
 
             throw new Exception($"FixedHttpClient: Unknown request '{requestUrl}'.");
         }
+
+        public int GetRequestCount()
```

Comment:
Why is this not a property?

Reply:
+1, would be easier to have it an internal / public property with private setter.